tensorboard/plugins/hparams/BUILD

```diff
@@ -45,7 +52,74 @@ py_test(
         "//tensorboard:expect_tensorflow_installed",
         "//tensorboard/backend/event_processing:event_accumulator",
         "//tensorboard/backend/event_processing:event_multiplexer",
-        "//tensorboard/util:tensor_util",
+    ],
+)
+
+py_test(
+    name = "list_metric_evals_test",
+    size = "small",
+    srcs = [
+        "list_metric_evals_test.py",
+    ],
+    deps = [
+        ":hparams_plugin",
+        "//tensorboard:expect_tensorflow_installed",
+    ],
+)
+
+py_test(
+    name = "backend_context_test",
+    size = "small",
+    srcs = [
+        "backend_context_test.py",
+    ],
+    deps = [
+        ":hparams_plugin",
+        "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/backend/event_processing:event_accumulator",
+        "//tensorboard/backend/event_processing:event_multiplexer",
+    ],
+)
+
+# To run the UI tests:
+#
+# bazel run //tensorboard/plugins/hparams:web_components_tests
+#
+# This will launch a webserver and print the port it listens on to stdout.
+# Open a browser and navigate to that port. Click on the
+# "/web_components_tests/web_components_tests.html" link. This will run the
+# tests and show the results in the browser.
+#
+# How to add a new component test:
+# 1. Add a 'test' subdir to the component dir: ./tf_hparams_<component>/test/
+#    and put the test in a a file named: tf-hparams-<component>-test.html in
+#    that directory.
+# 2. Add a 'test_lib' tf_web_library rule to the BUILD file of the component
+#    that contains the test.
+# 3. Update the 'web_components_tests.html' file to load
+#    '../tf-hparams-<component>/test/tf-hparams-<component>-test.html"
+# 4. Add "//tensorboard/plugins/hparams/tf_hparams_<component>:test_lib"
+#    as a dependency to the rule below.
+#
+# TODO(erez): Simplify adding a new test. It should not require touching so many files.
```

Comment:
Something like https://github.com/tensorflow/tensorboard/blob/1bfc270d81fa78730cf26d1bf1e98b1bf20a87ae/tensorboard/components/tf_backend/test/BUILD#L6-L14 should work

Reply:
Hmm. So you now have web_testing in opensource. Cool. I just want to make sure these tests will run in google3 as well and it seems like the tf_web_test does nothing in google3, currently. Let me think about it some more. Meanwhile, can you continue to look over this PR and let me know if you have any other comments?